README.md

```
@@ -0,0 +1,116 @@
+
+---
+
+### 4. Run the server
+
+```bash
+npm start
+```
+
+Server runs on:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)